### PR TITLE
chore(docs): update story14 to match result

### PR DESCRIPTION
**Pull request by ekeren**
*By submitting this pull request, I confirm that my contribution is made under the terms of the [Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

File: docs/06-contributors/999-rfcs/000-stories/story-14.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ let tasks = new TaskList();
 new cloud.Function(inflight (s: str): str => {
   tasks.add_task("clean the dishes");
   let result = tasks.find_tasks_with("clean the dishes");
-  assert(result.len == 1);
+  assert(result.length == 1);
   let t = tasks.get_task(result.at(0));
   assert("clean the dishes" == str.from_json(t.title));
```
Comment (staycoolcall911) [inline docs/06-contributors/999-rfcs/000-stories/story-14.md:135]:
```suggestion
  assert("clean the dishes" == str.from_json(t.get("title")));
```

File: docs/06-contributors/999-rfcs/000-stories/story-14.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ resource TaskList {
    */
   inflight add_estimation(id: str, effort_estimation: duration): str {
     let j = Json.clone_mut(this.get_task(id));
-    j.set("effort_estimation", effort_estimation);
+    j.set("effort_estimation_in_seconds", effort_estimation.seconds);
```
Comment (staycoolcall911) [inline docs/06-contributors/999-rfcs/000-stories/story-14.md:74]: We're currently unable to use duration literals in inflight: [Unable to use duration literals in inflights](https://github.com/winglang/wing/issues/1629)

Comment (ekeren) [reply docs/06-contributors/999-rfcs/000-stories/story-14.md:74]: Did we decide not to put this in the sprint?

Comment (staycoolcall911) [reply docs/06-contributors/999-rfcs/000-stories/story-14.md:74]: I think so. @Chriscbr is currently checking if the issue above is an easy fix. If not, we'll remove it from this story

Comment (staycoolcall911) [reply docs/06-contributors/999-rfcs/000-stories/story-14.md:74]: It'll take us time to get to add duration. I think we should remove story 14 altogether as it doesn't add much at this point.

**Comment by ekeren** [comment]
story 14 was deleted

**Pull request closed without merging**